### PR TITLE
Add signout feature

### PR DESCRIPTION
**Pull request by yenmai477**
I had handled signout feature. Please review it @no197

### CODE REVIEW

File: signout.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+sigout
```
Comment (no197) [inline signout.txt:1]: It's not correct. Please edit and make another request. Remember to capitalize first letter plz

Comment (yenmai477) [reply signout.txt:1]: Oh sorry, I'll fix it

File: signout.txt [hunk 2]
```diff
@@ -1 +1 @@
-sigout
+signout
```
Comment (no197) [inline signout.txt:1]: It's better but you should capitalize first letter

Comment (no197) [inline signout.txt:1]: It's better but you should capitalize a first letter

**Review by no197 — CHANGES_REQUESTED** [review]

File: signout.txt [hunk 3]
```diff
@@ -1 +1 @@
-signout
+SignOut
```
Comment (no197) [inline signout.txt:1]: It's perfect. Good job!

**Review by no197 — APPROVED** [review]

**Pull request merged**